scripts/transfer-issues.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+COUNT=1
```

Comment:
I assume you are planning to run with COUNT=10000 or something super high ? Or do you want to run this one by one in a loop ? Not sure what's better, I wonder how GitHub will cope with 7500 transfer requests... Have you asked our GitHub admins for guidance on this just in case?

(I tried `gh issue list -s all -L 10000 --json number` and it seems to be working FWIW, it's just a bit slow)

Reply:
Yeah, I'm not sure. I would definitely start with small batches. We have to consider not just the github side but also Jira and "unito" the tool we use to sync them. If I drop 10k tickets on addons, unito has to ingest them and create those issues in jira.

So many things could explode there.

FWIW, I did test what happens if you run the list command, and then transfer the issue and run the list again. It is not returned from the list after it is transferred so we can consider the script idempotent to a certain extent.